omc3/harpy/frequency.py

```diff
@@ -112,10 +112,16 @@ def harpy_per_plane(harpy_input, bpm_matrix, usv, tunes, plane):
     bpm_matrix = bpm_matrix.loc[df.index]
     spectra = {"FREQS": frequencies.loc[df.index], "COEFFS": coefficients.loc[df.index]}
 
-    if _get_natural_tunes(harpy_input, tunes) is not None:
-        df_nattunes = _calculate_natural_tunes(
-            spectra, _get_natural_tunes(harpy_input, tunes), harpy_input.tolerance, plane
-        )
+    nattunes = _get_natural_tunes(harpy_input, tunes)
+    tune_tol = harpy_input.tolerance
+    if nattunes is not None:
+        # Each plane that has 0 for the nattunes is ignored.
+        if any(abs(t - n) < tune_tol for n, t in zip(nattunes, tunes) if n != 0):
```

Comment:
I'm annoying and I'd prefer tune / nattune instead of t / n for variable names here but I'll leave it up to you.